integration/util/util.go

```diff
@@ -692,3 +694,27 @@ func NewPrometheus() *e2e.HTTPService {
 		9090,
 	)
 }
+
+type OtlpJaegerClient struct {
```

Comment:
What do you think if we move this into 

`github.com/grafana/tempo/pkg/util`

That way, we don't need to reference the integration package from Vulture.